### PR TITLE
fix(csharp): enforce dotnet sdk version

### PR DESCRIPTION
**Pull request by morganleroi**
## 🧭 What and Why

Enforce use of dotnet SDK specific version when using the CLI (CI).

### CODE REVIEW

**Review by shortcuts — APPROVED** [review]

File: generators/src/main/java/com/algolia/codegen/AlgoliaCSharpGenerator.java [hunk 1]
```diff
@@ -41,10 +45,20 @@ public String getClientName(String client) {
...
     CLIENT = (String) additionalProperties.get("client");
+
+    Stream<String> stream = null;
+    try {
+      stream = Files.lines(Paths.get("config/.csharp-version"));
+    } catch (IOException e) {
+      e.printStackTrace();
+      System.exit(1);
+    }
+
```
Comment (millotp) [inline generators/src/main/java/com/algolia/codegen/AlgoliaCSharpGenerator.java:56]: maybe just `Files.readString(fileName);`, and remove trailing `\n`, as you want

**Review by millotp — COMMENTED** [review]
thanks for finding a fix so quickly !

**Review by shortcuts — COMMENTED** [review]
can you edit https://github.com/algolia/api-clients-automation/blob/main/scripts/buildClients.ts#L23-L26 to be `find .` to avoid searching `cwd` in `cwd `pls?

**Review by shortcuts — DISMISSED** [review]
gg guys

**Review by millotp — APPROVED** [review]

**Pull request merged**